src/v0/destinations/salesforce/utils.js

```diff
@@ -23,6 +23,13 @@ const {
 
 const ACCESS_TOKEN_CACHE = new Cache(ACCESS_TOKEN_CACHE_TTL);
 
+const getErrorMessage = (response) => {
```

Comment:
Lets document the format of error messages here with reference

Reply:
done